### DISCUSSION TITLE
Duplicates Tab

### DISCUSSION THREAD

**Discussion by ccharlot** [nextcloud/bookmarks#2241] [Q&A]
Hi !,

I have a question about the Duplicates tab : 

I installed Nextcloud Bookrmarks and got my bookmarks synced with Floccus. Everything was ok.

Bu. I saw I had a lot of duplicates, so instead of removing them one by one in the app, I used another tool to remove them 'outside' of the Bookmark app.

Now I still have all duplicates still in the tab... and can't remove them from there, since that would completely delete them.

Anyone know how I could "reset" the duplicate tab ?

**Comment by marcelklehr** [comment]
> Anyone know how I could "reset" the duplicate tab ?

There should be no need to reset the duplicates tab. It will auto-update.
It may be that your other tool only removed duplicates in the same folder, while the app displays duplicates across your whole collection.

**Comment by ccharlot** [comment]
OK, I think I know what happened.

I had most of my duplicates already removed and in the trash.

After emptying the trash, the duplicate count and list is now OK.

So, maybe there is another issue there : maybe the app shouldn't count what is in the trash as duplicates ?

**Reply by marcelklehr** [reply]
Ah, that's a nice finding! I will look into this. Can you open a new issue for this so it doesn't get lost?
